src/storage/src/hummock/local_version_manager.rs

```diff
@@ -947,12 +888,6 @@ impl LocalVersionManager {
 #[cfg(any(test, feature = "test"))]
 // Some method specially for tests of `LocalVersionManager`
 impl LocalVersionManager {
-    pub async fn refresh_version(&self, hummock_meta_client: &dyn HummockMetaClient) -> bool {
```

Comment:
I think refresh_version is necessary for some tests? And we need reimplement refresh_version.

Reply:
With something like wait_version(version_id)?

Reply:
just resumed tests.

Reply:
> I think refresh_version is necessary for some tests? And we need reimplement refresh_version.

just resumed tests